src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -415,10 +411,71 @@ function createModel(program: Program, generateConvenienceAPI: boolean = false):
         }
     }
 
+    function emitClient(
+        client: Client | OperationGroup,
+        parent?: Client
+    ): InputClient {
+        const operations = listOperationsInOperationGroup(program, client);
+        let clientDesc = "";
+        if (operations.length > 0) {
+            const container = ignoreDiagnostics(
+                getHttpOperation(program, operations[0])
+            ).container;
+            clientDesc = getDoc(program, container) ?? "";
+        }
+
+        const inputClient = {
+            Name: client.kind === ClientKind.DpgClient ? client.name : client.type.name,
+            Description: clientDesc,
+            Operations: [],
+            Protocol: {},
+            Creatable: client.kind === ClientKind.DpgClient,
+            Parent: parent?.name
+        } as InputClient;
+        for (const op of operations) {
+            if (lroMonitorOperations.has(op)) continue;
+            const httpOperation = ignoreDiagnostics(
+                getHttpOperation(program, op)
+            );
+            const inputOperation: InputOperation = loadOperation(
+                program,
+                httpOperation,
+                url,
+                urlParameters,
+                modelMap,
+                enumMap
+            );
+
+            applyDefaultContentTypeAndAcceptParameter(inputOperation);
+
+            const apiVersionInOperation = inputOperation.Parameters.find(
+                isApiVersionParameter
+            );
+            if (apiVersionInOperation) {
```

Comment:
Combine  those 2 lines together?

Reply:
451 and 452? No. The prettier sperate to two lines.

Reply:
I mean 

if(apiVersionInOperation?.DefaultValue?.Value)